HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_02.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Homework3
+  CONDITION = %r{\S+\s-\s-\s\D\d+/\w+/\S+\s\+\S+\s\S+\s\S+\s\w+/\d\.\d"\s\d+\s\S+\s\S+}
+  DATETIME = %r{\d{2}/\w{3}/\d{4}:\d{2}:\d{2}:\d{2}\s\+\d{4}}
+  IP = /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/
+  ADDRESS = %r{/\w*/\d*/\w*}
+  def task2(log)
+    rows = log.split("\n").select { |str| str.include?('POST') }
+    rows.map do |element|
+      return [] unless element == element.scan(CONDITION).join
```

Comment:
better to use `select` instead of this condition inside the map 
```suggestion
    rows.select { _1 == _1.scan(CONDITION).join }.map do |element|
```